release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -117,6 +117,57 @@ The Dashboards view is now available from the Monitoring section in the web
 console. This lets you view metrics that bring transparency to the
 {product-title} cluster and its dependent components.
 
+[id="ocp-4-4-hwmon-collector-disabled"]
+==== hwmon collector disabled in node-exporter
+
+The `hwmon` collector has been disabled in the node-exporter monitoring
+component because it is no longer used to collect cluster metrics.
+
+[id="ocp-4-4-cluster-reader-can-read-node-metrics"]
+==== cluster-reader can read node metrics
+
+The `cluster-reader` role can now read node metrics by default.
+
+[id="ocp-4-4-new-cluster-alert"]
+==== Cluster alert for when multiple containers are killed
+
+When multiple containers have been killed within the past 15 minutes, you are
+now notified with a `MultipleContainersOOMKilled` alert.
+
+[id="ocp-4-4-new-api-server-alerts"]
+==== New API server alerts
+
+There are two new API server alerts available for {product-title} 4.4:
+
+* `ErrorBudgetBurn`: fires when the API server issues 5xx request responses.
+* `AggregatedAPIErrors`: fires when the number of errors have increased for the
+aggregated API servers.
+
+[id="ocp-4-4-permission-updates-for-po"]
+==== Permission updates for Prometheus Operator
+
+The custom resource definitions managed by the Prometheus Operator now have more
+restrictive permissions.
+
+The custom resources the Prometheus Operator manages include:
+
+* `Prometheus`
+* `ServiceMonitor`
+* `PodMonitor`
+* `Alertmanager`
+* `PrometheusRule`
+
+[id="ocp-4-4-cluster-monitoring-version-updates"]
+==== Cluster monitoring component version updates
+
+The following monitoring components have been upgraded:
+
+* Prometheus: version upgrade from 2.14.0 to 2.15.2
+* Alertmanager: version upgrade from 0.19.0 to 0.20.0
+* Prometheus Operator: version upgrade from 0.34.0 to 0.35.1
+* `kube-state-metrics`: version upgrade from 1.8.0 to 1.9.5
```

Comment:
Why is kube-state-metrics in code style ticks?

Reply:
I'll remove them for consistency with the others. I had seen them with backticks elsewhere in the docs, but it seems the majority are without.